bots/images/scripts/ovirt.setup

```diff
@@ -0,0 +1,151 @@
+
+# Latest stable repos
+rpm -ivh http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-ovirt-*
+rpm --import https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
+
+# Install ovirt-engine
+yum install -y ovirt-engine-setup
+echo 'SSO_ALTERNATE_ENGINE_FQDNS="localhost:443"' >> /etc/ovirt-engine/engine.conf.d/999-ovirt-engine.conf
+cat <<EOF > /answers.conf
+[environment:default]
+OVESETUP_DB/user=str:engine
+OVESETUP_DB/password=str:engine
+OVESETUP_DB/database=str:engine
+OVESETUP_DB/host=str:${FQDN}
+OVESETUP_DB/port=str:5432
+OVESETUP_ENGINE_CONFIG/fqdn=str:${FQDN}
+OVESETUP_CONFIG/fqdn=str:${FQDN}
+OVESETUP_CONFIG/adminPassword=str:engine
+OVESETUP_PKI/organization=str:oVirt
+
+OVESETUP_DIALOG/confirmSettings=bool:True
+OVESETUP_CONFIG/applicationMode=str:both
+OVESETUP_CONFIG/remoteEngineSetupStyle=none:None
+OVESETUP_CONFIG/sanWipeAfterDelete=bool:False
+OVESETUP_CONFIG/storageIsLocal=bool:False
+OVESETUP_CONFIG/firewallManager=none:None
+OVESETUP_CONFIG/remoteEngineHostRootPassword=none:None
+OVESETUP_CONFIG/firewallChangesReview=none:None
+OVESETUP_CONFIG/updateFirewall=bool:False
+OVESETUP_CONFIG/remoteEngineHostSshPort=none:None
+OVESETUP_CONFIG/storageType=none:None
+OSETUP_RPMDISTRO/requireRollback=none:None
+OSETUP_RPMDISTRO/enableUpgrade=none:None
+OVESETUP_DB/upgradeWithHeEl6Hosts=none:None
+OVESETUP_DB/secured=bool:False
+OVESETUP_DB/dumper=str:pg_custom
+OVESETUP_DB/fixDbViolations=none:None
+OVESETUP_DB/filter=none:None
+OVESETUP_DB/restoreJobs=int:2
+OVESETUP_DB/securedHostValidation=bool:False
+OVESETUP_ENGINE_CORE/enable=bool:True
+OVESETUP_CORE/engineStop=none:None
+OVESETUP_SYSTEM/memCheckEnabled=bool:False
+OVESETUP_SYSTEM/nfsConfigEnabled=none:None
+OVESETUP_PKI/renew=none:None
+OVESETUP_CONFIG/isoDomainName=none:None
+OVESETUP_CONFIG/engineHeapMax=str:2048M
+OVESETUP_CONFIG/isoDomainACL=none:None
+OVESETUP_CONFIG/isoDomainMountPoint=none:None
+OVESETUP_CONFIG/engineDbBackupDir=str:/var/lib/ovirt-engine/backups
+OVESETUP_CONFIG/engineHeapMin=str:2048M
+OVESETUP_AIO/configure=none:None
+OVESETUP_AIO/storageDomainName=none:None
+OVESETUP_AIO/storageDomainDir=none:None
+OVESETUP_PROVISIONING/postgresProvisioningEnabled=bool:True
+OVESETUP_APACHE/configureRootRedirection=bool:True
+OVESETUP_APACHE/configureSsl=bool:True
+OVESETUP_VMCONSOLE_PROXY_CONFIG/vmconsoleProxyConfig=bool:True
+OVESETUP_CONFIG/websocketProxyConfig=bool:True
+OVESETUP_CONFIG/fenceKdumpListenerStopNeeded=bool:False
+OVESETUP_APACHE/enable=bool:False
+OVESETUP_CONFIG/engineServiceStopNeeded=bool:False
+OVESETUP_ASYNC/clearTasks=bool:False
+OVESETUP_DWH_CORE/enable=bool:False
+OVESETUP_DWH_PROVISIONING/postgresProvisioningEnabled=bool:False
+OVESETUP_CONFIG/imageioProxyConfig=bool:False
+# Don't install the the OVN provider:
+OVESETUP_OVN/ovirtProviderOvn=bool:False
+# Don't perform database full vacuum:
+OVESETUP_DB/engineVacuumFull=bool:False
+EOF
```

Comment:
most of them are needed. The setup is typically manual. For reference I took it from https://github.com/oVirt/ovirt-containers/tree/master/image-specifications/engine which is the closest thing we have to unattended preconfigured instance.
I can clean something out knowing that we won't use those features in cockpit CI...but most of them will stay. 
Note it is a bit future-proof with the --accept-defaults option already.